container/docker/factory.go

```diff
@@ -284,12 +286,18 @@ func Register(factory info.MachineInfoFactory, fsInfo fs.FsInfo, ignoreMetrics c
 		return fmt.Errorf("failed to get cgroup subsystems: %v", err)
 	}
 
-	var thinPoolWatcher *devicemapper.ThinPoolWatcher
+	var (
```

Comment:
You must have a background in C :-p

On Thursday, August 25, 2016, Timothy St. Clair <notifications@github.com>
wrote:

> In container/docker/factory.go
> <https://github.com/google/cadvisor/pull/1439#discussion_r76323572>:
>
> > @@ -284,12 +286,18 @@ func Register(factory info.MachineInfoFactory, fsInfo fs.FsInfo, ignoreMetrics c
> >  		return fmt.Errorf("failed to get cgroup subsystems: %v", err)
> >  	}
> >
> > -	var thinPoolWatcher *devicemapper.ThinPoolWatcher
> > +	var (
>
> could var-block at the top for readability.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/cadvisor/pull/1439/files/27b9eed7388c7627f53143e9be16112488dffa28#r76323572>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAABYvxhPJR4gq_OpLvxLeVhvUXpLQXgks5qjgKGgaJpZM4Jtfne>
> .
>